packages/aws-cdk-lib/aws-globalaccelerator/lib/accelerator.ts

```diff
@@ -82,6 +142,18 @@ export class Accelerator extends cdk.Resource implements IAccelerator {
    */
   public readonly dnsName: string;
 
+  /**
+   * IP addressess associated with the accelerator.
+   */
+  public readonly ipAddresses?: string[];
+
+  /**
+   * The IP address type that an accelerator supports.
+   *
+   * For a standard accelerator, the value can be IPV4 or DUAL_STACK.
+  */
+  public readonly ipAddressType?: IpAddressType;
+
   constructor(scope: Construct, id: string, props: AcceleratorProps = {}) {
     super(scope, id);
```

Comment:
Like the `validateAcceleratorName` method, it would be good to validate whether the number of elements in IpAddresses is not greater than two in the constructor.

> You can specify one or two addresses, separated by a comma.

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-globalaccelerator-accelerator.html#aws-resource-globalaccelerator-accelerator-properties